.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,4 +1,4 @@
-blank_issues_enabled: true
+blank_issues_enabled: false
```

Comment:
Why do we want to disable blank issues? It's sometimes useful if the issue doesn't match one of the templates.